docs/services/zwo_camera.rst

```diff
@@ -1,14 +1,64 @@
 ZWO Camera
 ==========
 
+This service operates a ZWO camera. There are currently four different types of ZWO cameras that have been tested and used with catkit2 so far:
```

Comment:
Ditch the "four" to limit the risk of incomplete list extension in the future, put down something like "the following devices" instead.